### ISSUE TITLE
Are we ever going to add manuals to AL? Or leave that to Atarimania?

### ISSUE THREAD

**Issue by MugUK** [atari-legend/atari-legend#229]
It's only because someone has sent in a manual for Night Raider to the database submissions.

**Comment by nguillaumin** [comment]
We could. I would tend to do like for the magazines and store them on archive.org ...

**Comment by MugUK** [comment]
If we've already done it for magazines, then it would be just as easy I guess to add a manual, especially as a lot of them are already up on archive.org. Would we just add a URL that points to the manual into CPanel for the game?

**Comment by nguillaumin** [comment]
Yes, I need to add a new field in the DB for this, and present it on the main site. I think it belongs to the release though, not the game. I will have a look soon.

**Comment by MugUK** [comment]
Definitely at release level as the budget releases will have had the original manuals compressed down to fit on less sheets of paper etc.
